tests/suites/test_suite_cipher.aes.data

```diff
@@ -474,6 +474,70 @@ AES-128 CFB - Encrypt and decrypt 32 bytes in multiple parts 1
 depends_on:MBEDTLS_AES_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf_multipart:MBEDTLS_CIPHER_AES_128_CFB128:128:16:16:-1:16:16:16:16
 
```

Comment:
Please add negative test cases with 0 bytes and 15 bytes to test error detection. Ideally we should also have a test with 2^24+1 bytes, if this can reasonably be done without allocating that much memory.

Reply:
I had a lot of the negative test cases you are describing already. I've added a test for 0 bytes, but already had tests for 15 bytes and 2^24 + 1 bytes. Please let me know if the current set of negative test cases is insufficient.